coding_systems/snomedct/coding_system.py

```diff
@@ -85,3 +85,11 @@ def descendant_relationships(codes):
     """
 
     return query(sql, codes)
+
+
+def get_code_to_term(codes):
```

Comment:
What's the difference between this and `lookup_names()` above?

Reply:
"programmer error" 😬  fixed in c5c1140